src/hyperion/device_setup_plans/setup_panda.py

```diff
@@ -25,10 +25,10 @@ class Enabled(Enum):
 
 
 def get_seq_table(
-    parameters: PandAGridScanParams, time_between_x_steps_ms, exposure_time_s
+    parameters: PandAGridScanParams,
+    exposure_distance_mm,
```

Comment:
Could: `exposure_distance` is probably a weird term at first glance, can you clarify it in the docstring?